pbr/util.py

```diff
@@ -286,9 +286,47 @@ def point_on_field(cam_location, mask_path, env_info, num_points):
 
     # Check if environment map has field points, else set to origin
     if len(field_coords) > 0:
-        for ii in range(num_points):
+        while len(ground_points) < scene_config.num_robots:
             # Get random field point
             y, x = field_coords[rand.randint(0, field_coords.shape[0] - 1)]
+            yproj, xproj = project_to_ground(y, x, cam_location, img, env_info)
+
+            if any(
+                [
+                    (p[0] - xproj) ** 2 + (p[1] - yproj) ** 2
+                    < scene_config.radius_robot**2
+                    for p in ground_points
+                ]
+            ):
+                print("Skipping point")
+                continue
 
             ground_points.append(project_to_ground(y, x, cam_location, img, env_info))
+
     return ground_points
+
+
+def generate_moves(field_obj, z_coord=0.3, radius=0.7):
```

Comment:
you should add a function description just above the function. description, what the arguments mean, what any return means.

Reply:
The function comment doesn't really describe the function, just how it's currently being used in the code somewhere? You should write the comment here that describes the function, then when you use it you can explain what it going on there. 
 
The function itself is one that builds a list of random position points on the field that are not close to each other, based on the given radius. 
`field_obj` gives the dimensions of the field for generating the points
`z_coord` is what, the height of the robot?? the torso?
`radius` is what, the distance points need to be away from each other? the distance between robots?

Reply:
see [19c590c](https://github.com/NUbots/NUpbr/pull/19/commits/19c590c7a2c4225f554eb5d5f90064ad9e4652c9)

and 

[7f83942](https://github.com/NUbots/NUpbr/pull/19/commits/7f839426c2b395ad8d8b74b3163ab81edef31bed)

for updated docs